src/test_db_connection.py

```diff
@@ -0,0 +1,37 @@
+import os
+import yaml
+import cx_Oracle
+from pathlib import Path
+home = str(Path.home())
+
+
+
+def process_yaml():
+	with open("../config.yaml") as file:
+		return yaml.safe_load(file)
+
+
+
+# wallet location (default is HOME/wallets/wallet_X)
+os.environ['TNS_ADMIN'] = '{}/{}'.format(home, 'wallets/Wallet_eSportsDB')
```

Comment:
Shouldn't the wallet name be in a variable, that's populated from an environment variable or a config file (rather than hard-coded)?

Reply:
Fixed in commit